experimental/examples/ConfigExample.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../../vendor/autoload.php';
+
+use OpenTelemetry\Experimental\Config\ConfigBuilder;
+use OpenTelemetry\Experimental\Config\Exporter\NewRelic\Config as NewRelicConfig;
+use OpenTelemetry\Experimental\Config\Exporter\Zipkin\Config as ZipkinConfig;
+
+//@see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md
+putenv('OTEL_RESOURCE_ATTRIBUTES=service.version=1.0.0,service.namespace=my_app');
+putenv('OTEL_SERVICE_NAME=example-app');
+putenv('OTEL_LOG_LEVEL=warning');
+putenv('OTEL_TRACES_SAMPLER=traceidratio');
+putenv('OTEL_TRACES_SAMPLER_ARG=0.95');
+putenv('OTEL_TRACES_EXPORTER=otlp,zipkin,newrelic');
+putenv('OTEL_EXPORTER_ZIPKIN_ENDPOINT=http:zipkin/v1/traces');
+putenv('OTEL_EXPORTER_ZIPKIN_TIMEOUT=99');
+putenv('OTEL_PHP_TRACES_PROCESSOR=batch,simple,noop');
+putenv('OTEL_BSP_SCHEDULE_DELAY=10000');
+putenv('OTEL_ATTRIBUTE_COUNT_LIMIT=111');
+putenv('OTEL_PROPAGATORS=tracecontext');
+
+echo 'Creating Config From Environment and user config' . PHP_EOL;
+$config = (new ConfigBuilder())
+    ->withExporterConfig(ZipkinConfig::class)
+    ->withExporterConfig(NewRelicConfig::class)
+    ->withUserConfig([
+        'span.processor.batch.max_queue_size' => 333,
+        'resource.limits.attribute_value_length' => 444,
+        'exporter.new_relic.license_key' => 'secret',
```

Comment:
So I can only have one newrelic exporter?

Reply:
It never occurred to me that somebody would want to have multiple instances of the same exporter. In fact, the Batch and Simple span processors only allow for 0..1 exporter. MultiSpanProcessor does look like it allows any number of span processors, each of which could have their own exporter though - so I guess you can!

Reply:
> It never occurred to me that somebody would want to have multiple instances of the same exporter.

I guess you never had to develop packages/libraries to be used by customers/clients. They sometimes can come up with let's say kind of "exotic" requests or requierements, that's why I'm used to at least consider "exotic" requirements to be prepared. ;)